srml/contract/src/rent.rs

```diff
@@ -0,0 +1,138 @@
+use crate::{BalanceOf, ContractInfo, ContractInfoOf, Module, TombstoneContractInfo, Trait};
+use runtime_primitives::traits::{As, Bounded, CheckedDiv, CheckedMul, Saturating, Zero};
+use srml_support::traits::{Currency, ExistenceRequirement, Imbalance, WithdrawReason};
+use srml_support::StorageMap;
+
+/// Evict and optionnaly pay rent, at block number. Return if evicted
```

Comment:
Returns **what** if evicted?

Reply:
+ typo

```suggestion
/// Evict and optionally pay rent, at block number. Return if evicted
```

Reply:
`return = if evicted { true } else { false }`
Ok I'll try to reformulate

Reply:
`Returns true if evicted` would work : )